Add LGBTQ stylebook

lucindakn opened the pull request:
@claude please rename this to lgbtq_stylebook.jinja2

lucindakn committed:
Add LGBTQ stylebook

nicsuzor commented:
@claude please take look at this:
[] rename the file to lgbtq_stylebook.jinja2
[] get rid of headings for each letter ('### A' etc)
[] Remove the 'introduction' heading and cut  lines 2-7

lucindakn commented:
add to top of file 
```
{# name: lgbtq_stylebook
# info: NLGJA: The Association of LGBTQ+ Journalists’ Stylebook on LGBTQ+ Terminology
# author: NLGJA: The Association of LGBTQ+ Journalists
# link: https://www.nlgja.org/stylebook-on-lgbtq-terminology/
# date-retrieved: 2025-05-08 #}
```

nicsuzor commented:
@claude did you commit these changes?

nicsuzor commented:
@claude ok just make the edits, don't rename the file

claude[bot] committed:
Update LGBTQ+ stylebook: add metadata header, remove intro and letter headings

- Added metadata header with source information
- Removed Introduction section and intro content
- Removed all alphabetical section headings (A, B, C, etc.)

Co-authored-by: Nicolas Suzor <nicsuzor@users.noreply.github.com>

nicsuzor commented:
@claude now rename this to lgbtq_stylebook.jinja2 (I've granted bash permissions)

claude[bot] committed:
Rename lgbtq_stylebook.jinja2.j2 to lgbtq_stylebook.jinja2

Co-authored-by: Nicolas Suzor <nicsuzor@users.noreply.github.com>
